_codux/ui-kit/ui-kit-utils/kit.tsx

```diff
@@ -0,0 +1,52 @@
+import classNames from 'classnames';
+import { ReactNode } from 'react';
+import styles from './kit.module.scss';
+
+interface KitProps {
+    category: string;
+    title: string;
+    children?: ReactNode;
+    className?: string;
+}
+
+export const Kit = ({ category, title, children, className }: KitProps) => {
+    return (
+        <div className={classNames(styles.container, className)}>
+            <div className={styles.header}>
+                <b>UI Kit</b> | {category}
+            </div>
+            <h3 className={styles.title}>{title}</h3>
```

Comment:
Itn't category already a title?

Reply:
Categories are 'Foundation' and 'Core components'. 
Titles are 'Typography', 'Buttons', 'Sections', 'Components & Elements'

Reply:
> Titles are 'Typography', 'Buttons', 'Sections', 'Components & Elements'

Aren't they titles of `<Kit.Section>`?

Reply:
Nope. 
Sometimes they are the same however.